composer.json

```diff
@@ -53,7 +53,7 @@
         "php": "^7.4 || ~8.0.0 || ~8.1.0",
         "ext-mongodb": "*",
         "container-interop/container-interop": "^1.2.0",
-        "doctrine/doctrine-module": "^5.0.0",
+        "doctrine/doctrine-module": "^5.0.0 <5.2.0",
```

Comment:
Shouldn't there be a comma here? :thinking:

Reply:
According to https://getcomposer.org/doc/articles/versions.md#version-range:

> You can define multiple ranges. Ranges separated by a space ( ) or comma (,) will be treated as a logical AND. A double pipe (||) will be treated as a logical OR. AND has higher precedence than OR.

So it shouldn't matter :-)